src/PhpStan/AbstractNoNamedArgumentUsedRule.php

```diff
@@ -70,11 +71,9 @@ public function processNode(Node $node, Scope $scope): array
 
                 if ('all' === $forbiddenMethods || \in_array(\strtolower($methodName), $forbiddenMethods, true)) {
                     if ($this->findNullableName($node)) {
-                        $errors[] = \sprintf(
-                            'The method "%s::%s" is forbidden to call without named arguments.',
-                            $className,
-                            $methodName
-                        );
+                        $errors[] = RuleErrorBuilder::message(\sprintf('The method "%s::%s" is forbidden to call without named arguments.', $className, $methodName))
+                            ->identifier('method.call.without.named.arguments.forbidden')
```

Comment:
Names of identifiers is very hard for understand and next setting in phpstan.neon.
We should group identifiers by specific names, and maybe you can see phpstan identifiers for correct names?